device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/SubscriptionMessageRequestSentCallback.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.azure.sdk.iot.device.transport.amqps;
+
+import com.microsoft.azure.sdk.iot.device.DeviceTwin.DeviceOperations;
+
+public interface SubscriptionMessageRequestSentCallback
+{
+    void onSubscriptionMessageSent(int deliveryTag, SubscriptionType deviceOperationType);
```

Comment:
subscriptionType instead of deviceOperationType?

Reply:
Good catch